modular_skyrat/modules/reagent_forging/code/forge_items.dm

```diff
@@ -84,6 +104,11 @@
 	icon_state = "hot_spearhead"
 	spawn_item = /obj/item/forging/complete/spear
 
+/obj/item/forging/incomplete/bokken
+	name = "incomplete bokken head"
+	icon_state = "hot_blade"
+	spawn_item = /obj/item/forging/complete/bokken
+
```

Comment:
bokken are _always_ wood, they're strictly training weapons. why is this a forging item? moreover, why does it look like a hot metal blade?